### ISSUE TITLE
rewrite index_fulltext_lexeme_update_trigger to not use string manip

### ISSUE THREAD

**Issue by reedstrm** [Connexions/cnx-archive#527]
The current trigger on modulefti that converts the tsvector into a set of lexemes in modulefti_lexemes uses string parsing to create the list of lexemes. This can now be done with ts_stat()

index_fulltext_lexeme_update_trigger, also index_collated_fulltext_lexeme_update_trigger
Need to verify that the 'positions' column is only used for counting

**Comment by reedstrm** [comment]
Some quick tests say it's 100x faster. Yes, two orders of magnitude.

**Comment by pumazi** [comment]
Traceback associated with this issue:

```pytb
Traceback (most recent call last):
  File "/Users/pumazi/.environments/cnx/bin/dbmigrator", line 11, in <module>
    load_entry_point('db-migrator', 'console_scripts', 'dbmigrator')()
  File "/Users/pumazi/cnx/src/db-migrator/dbmigrator/cli.py", line 104, in main
    return args['cmmd'](**args)
  File "/Users/pumazi/cnx/src/db-migrator/dbmigrator/utils.py", line 145, in wrapper
    return func(cursor, *args, **kwargs)
  File "/Users/pumazi/cnx/src/db-migrator/dbmigrator/commands/migrate.py", line 32, in cli_command
    run_deferred)
  File "/Users/pumazi/cnx/src/db-migrator/dbmigrator/utils.py", line 227, in compare_schema
    callback(*args, **kwargs)
  File "/Users/pumazi/cnx/src/db-migrator/dbmigrator/utils.py", line 257, in run_migration
    migration.up(cursor)
  File "../src/cnx-db/cnxdb/migrations/20170911201035_transform_cnxml_to_html.py", line 102, in up
    WHERE fileid = %s""", (new_fileid[0][0], fileid))
psycopg2.DataError: negative substring length not allowed
CONTEXT:  SQL statement "INSERT into modulefti_lexemes (module_ident, lexeme, positions)
        (with lex as (SELECT regexp_split_to_table(NEW.module_idx::text, E' \'') as t )
        SELECT NEW.module_ident,
               substring(t,1,strpos(t,E'\':')-1),
               ('{'||substring(t,strpos(t,E'\':')+2)||'}')::int[] from lex)"
PL/pgSQL function index_fulltext_lexeme_update_trigger() line 6 at SQL statement
SQL statement "UPDATE modulefti SET (fulltext, module_idx) = ( _baretext, _idx_vectors )
        WHERE module_ident = NEW.module_ident"
PL/pgSQL function index_fulltext_trigger() line 15 at SQL statement
```

**Comment by reedstrm** [comment]
closed this by removing the table completely, and getting the lexeme count a different way. See: https://github.com/Connexions/cnx-db/blob/b1c14f19c38a2324311ca2e19826f8a73d006270/cnxdb/archive-sql/schema/fulltext-indexing.sql#L40-L48  Closed by Connexions/cnx-db/pull/78
